app/controllers/api/v1/court_hearings_controller.rb

```diff
@@ -31,11 +31,17 @@ def court_hearings_params
       end
 
       def move
-        id = params.require(:data).dig(:relationships, :move, :data, :id)
+        @move ||= begin
+                    id = params.require(:data).dig(:relationships, :move, :data, :id)
 
-        return if id.blank?
+                    if id.present?
+                      Move.find(id)
+                    end
```

Comment:
is not the same as `return if id.blank?`

Reply:
It is, yeah :).